src/resmom/start_exec.c

```diff
@@ -2168,8 +2177,9 @@ receive_string_data(int downfds, int  upfds)
 	}
 	memcpy(&r_size, r_buf, sizeof(size_t));
 	/* ack that we got the r_size */
-	if (write_pipe_data(upfds, &r_size, sizeof(size_t)) != 0) {
-		snprintf(log_buffer, sizeof(log_buffer), "write of %lu to pipe failed", r_size);
+	ack_size = sizeof(size_t);
+	if (write_pipe_data(upfds, &ack_size, sizeof(size_t)) != 0) {
+		snprintf(log_buffer, sizeof(log_buffer), "write of %lu to pipe failed", ack_size);
```

Comment:
How about... write of length 1234 bytes to pipe failed

Reply:
Sure. Modified.